### PR TITLE
Update remaining updateeachrelease docs

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #34171

---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/deploying/creating-nuget-packages.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/core/deploying/creating-nuget-packages.md) | [How to create a NuGet package with the .NET CLI](https://review.learn.microsoft.com/en-us/dotnet/core/deploying/creating-nuget-packages?branch=pr-en-us-38267) |
| [docs/core/deploying/deploy-with-cli.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/core/deploying/deploy-with-cli.md) | [Publish .NET apps with the .NET CLI](https://review.learn.microsoft.com/en-us/dotnet/core/deploying/deploy-with-cli?branch=pr-en-us-38267) |
| [docs/core/install/how-to-detect-installed-versions.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/core/install/how-to-detect-installed-versions.md) | [How to check that .NET is already installed](https://review.learn.microsoft.com/en-us/dotnet/core/install/how-to-detect-installed-versions?branch=pr-en-us-38267) |
| [docs/core/install/linux-ubuntu.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/core/install/linux-ubuntu.md) | [Get Ubuntu version](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu?branch=pr-en-us-38267) |
| [docs/core/introduction.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/core/introduction.md) | [.NET (and .NET Core) - introduction and overview](https://review.learn.microsoft.com/en-us/dotnet/core/introduction?branch=pr-en-us-38267) |
| [docs/core/porting/versioning-sdk-msbuild-vs.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/core/porting/versioning-sdk-msbuild-vs.md) | [.NET SDK, MSBuild, and Visual Studio versioning](https://review.learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs?branch=pr-en-us-38267) |
| [docs/core/project-sdk/msbuild-props-desktop.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/core/project-sdk/msbuild-props-desktop.md) | [MSBuild reference for .NET Desktop SDK projects](https://review.learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props-desktop?branch=pr-en-us-38267) |
| [docs/core/project-sdk/msbuild-props.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/core/project-sdk/msbuild-props.md) | [docs/core/project-sdk/msbuild-props](https://review.learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props?branch=pr-en-us-38267) |
| [docs/core/sdk.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/core/sdk.md) | [What is the .NET SDK?](https://review.learn.microsoft.com/en-us/dotnet/core/sdk?branch=pr-en-us-38267) |
| [docs/fundamentals/code-analysis/overview.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/fundamentals/code-analysis/overview.md) | [Overview of .NET source code analysis](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/overview?branch=pr-en-us-38267) |
| [docs/fundamentals/implementations.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/fundamentals/implementations.md) | [.NET implementations](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/implementations?branch=pr-en-us-38267) |
| [docs/standard/net-standard.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/standard/net-standard.md) | [.NET Standard](https://review.learn.microsoft.com/en-us/dotnet/standard/net-standard?branch=pr-en-us-38267) |
| [docs/welcome.md](https://github.com/dotnet/docs/blob/9f5f7aa15766c026a28ce2bb1b69286247e9a00c/docs/welcome.md) | [docs/welcome](https://review.learn.microsoft.com/en-us/dotnet/welcome?branch=pr-en-us-38267) |

</details>

### CODE REVIEW

File: docs/core/deploying/deploy-with-cli.md [hunk 1]
```diff
@@ -18,39 +18,26 @@ For an overview of these publishing modes, see [.NET Application Deployment](ind
...
-|                                                                   | 3.1         | `dotnet publish -c Release -r <RID> --self-contained true`                                 |
-|                                                                   | 5.0         | `dotnet publish -c Release -r <RID> --self-contained true`                                 |
-|                                                                   | 6.0         | `dotnet publish -c Release -r <RID> --self-contained true`                                 |
-|                                                                   | 7.0         | `dotnet publish -c Release -r <RID> --self-contained true`                                 |
-
-\* When using **SDK version 3.1** or higher, framework-dependent executable is the default publishing mode when running the basic `dotnet publish` command.
+| Publish Mode                                                      | Command                                                                                    |
```
Comment (gewarren) [inline docs/core/deploying/deploy-with-cli.md:21]: I dropped the versions that are out of support @adegeo.

**Comment by shanselman** [comment]
This should be updated also, just installing dotnet 8 from Ubuntu. And maybe add metadata so we get it for 9 next time. https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#register-the-microsoft-package-repository

File: docs/fundamentals/implementations.md [hunk 2]
```diff
@@ -10,25 +10,25 @@ A .NET app is developed for one or more *[implementations](../standard/glossary.
...
+- .NET 6 and later versions
 - .NET Framework
 - Mono
 - UWP
 
-.NET 7 is currently the primary implementation, and the one that's the focus of ongoing development. .NET 7 is built on a single code base that supports multiple platforms and many workloads, such as Windows desktop apps and cross-platform console apps, cloud services, and websites. [Some workloads](../core/whats-new/dotnet-6.md#sdk-workloads), such as .NET WebAssembly build tools, are available as optional installations.
+.NET, previously referred to as .NET Core, is currently the primary implementation. .NET (8) is built on a single code base that supports multiple platforms and many workloads, such as Windows desktop apps and cross-platform console apps, cloud services, and websites. [Some workloads](../core/whats-new/dotnet-6.md#sdk-workloads), such as .NET WebAssembly build tools, are available as optional installations.
 
-## .NET 5 and later versions
+## .NET 6 and later versions
```
Comment (BillWagner) [inline docs/fundamentals/implementations.md:27]: Should this one stay as ".NET 5 and later"? Or did you update it because 5 is out of support?

**Review by BillWagner — APPROVED** [review]
This LGTM.

**Pull request merged**